### DISCUSSION TITLE
Why aren't the relase notes being updated....

### DISCUSSION THREAD

**Discussion by TheDirkMelvin** [netbox-community/netbox#7902] [General]
I was running Netbox 3.0.7, since its release in October. I have had no indicators that there was an update available, and when I check the 'official?' release notes page, it still shows 3.0.7 as the latest release. Today I realized, the when I click on the 'docs' link at the bottom of our Netbox site, it is taking me to a LOCAL version of the Netbox site. So this is sort of misleading to me. Has this always been the case?

Anyway, I ran the update and now we are on v3.0.10.

**Comment by jeremystretch** [comment]
A local build of the documentation started being included in NetBox v3.0 (see #6328). The documentation for the current stable release is [published on ReadTheDocs](https://netbox.readthedocs.io/en/stable/). You can also enable automatic checking for new releases by setting [`RELEASE_CHECK_URL`](https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#release_check_url).

**Reply by TheDirkMelvin** [reply]
Thank you for the swift response.
